zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -1648,6 +1648,11 @@ New features that are currently considered experimental.
     values and see changes from other clients. See
     ZOOKEEPER-784 for more details.
 
+* *zookeeper.follower.skipLearnerRequestToNextProcessor* :
+    (Java system property: **zookeeper.follower.skipLearnerRequestToNextProcessor**)
+    Boolean flag to skip learner request to be processed by next processor of followerRequestProcessor when set to true.
+    Default value is false.
+
```

Comment:
we should improve this description, this is a bit too technical, the users will not understand it without understanding a very specific small part in the code. We should mention that this property helps when there are observers used in the cluster. Also we can mention that it helps in memory pressure.

Reply:
I agree. @mayank99fb Please add some clarification in what circumstances could this setting be useful.

Reply:
@symat i'll update the description

Reply:
cool, I like